### DISCUSSION TITLE
tables created with tbl_summary package don't display correctly in PDF render

### DISCUSSION THREAD

**Discussion by rubinjm** [quarto-dev/quarto-cli#717] [Q&A]
I am having a problem rendering a PDF document with tables created with the tbl_summary package. They display properly when rendered to HTML, though. 

Below is a sample of the R code along with examples of how it displays correctly when rendered to HTML (fonts, subtitle with min and max dates, formatting) as compared to the PDF example. I've tried multiple PDF engines and they all look the same. Am I missing something?

**R code-**
`t1 = ctpe.data %>%
  select(site, result) %>%
  tbl_summary(
    label = list(
      site ~ "Location",
      result ~ "Result"
    )
  ) %>%
  bold_labels() %>%
  as_gt ()%>%
  opt_row_striping() %>%
  opt_table_lines("none") %>%
  tab_header(
    title = md("**CTPE scans ordered in XXX**"),
    subtitle = c("Includes scans performed ", format(min(cleaned.data$date_time),"%m/%d/%y"), "-", format(max(cleaned.data$date_time),"%m/%d/%y"))
  ) %>%
  opt_table_font(
    font = google_font("Roboto Condensed"),
    weight = 500
  )

`

**PDF sample-**
<img width="631" alt="PDF output" src="https://user-images.githubusercontent.com/48840674/164780626-8fbe59ec-2729-4e5a-bda9-c25c880818d0.png">

**HTML sample-**
<img width="741" alt="HTML output" src="https://user-images.githubusercontent.com/48840674/164780637-604720b1-a19c-4c54-ba6c-659197be4bbf.png">

**Comment by cderv** [comment]
By `tbl_summary()` you are talking about the function in **gtsummary** package ? 

Can you share a reproducible example that I could run on my side ? Which package you are using and which dataset ?  I don't know how to find `ctpe.data`. 

Also, @rich-iannone is looking to improve **gt** outputs for Quarto document, maybe this would help the **gtsummary** package that I think is using **gt** as backend.
